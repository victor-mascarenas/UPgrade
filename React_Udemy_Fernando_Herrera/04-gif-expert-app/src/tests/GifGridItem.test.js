import React from "react";
import {shallow} from 'enzyme';
import GifGridItem from '../components/GifGridItem';

describe('Pruebas en GifGridItem', () => {
    const title = 'Un titulo';
    const url = 'https://localhost:8080/img/1.gif'
    test('Debe mostrar el componente correctamente', () => {
        const wrapper = shallow(<GifGridItem title={title} url={url}/>);
        expect(wrapper).toMatchSnapshot();
    });
});
