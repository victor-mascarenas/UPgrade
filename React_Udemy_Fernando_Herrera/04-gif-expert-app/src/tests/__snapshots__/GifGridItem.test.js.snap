// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifGridItem Debe mostrar el componente correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Un titulo"
    src="https://localhost:8080/img/1.gif"
  />
  <p>
    Un titulo
  </p>
</div>
`;
